lib/inspec/resources/sybase_session.rb

```diff
@@ -64,6 +64,10 @@ def query(sql)
       DatabaseHelper::SQLQueryResult.new(isql_cmd, parse_csv_result(isql_cmd.stdout))
     end
 
+    def resource_id
+      "Sybase Session"
```

Comment:
Maybe we can use username, db or some other combination here right? These are defined in initialiser.

Reply:
Those values can be sensitive information

Reply:
Except password, user name or db name can be used I feel. Those might not be that sensitive for creating resource id.

Reply:
Changed accordingly to consider the database.